Documentation/docs/learn/faq.md

```diff
@@ -134,13 +134,10 @@ in case of desperation.
 1. Write a MetaImage header. This is a small text file holding
    information about the image: spacing, dimensions, pixelt type, etc.
    This header can hold the list of DICOM files you want to read. The
-   only restriction is that the files must be uncompressed.
-   The easy way to create a MetaImage header is to use the application:
-   `MetaImageImporter` that is available in
-   [ITKApps/MetaImageImporter](https://github.com/InsightSoftwareConsortium/ITKApps/tree/master/MetaImageImporter).
-   You can also take an existing MetaImage reader and modify it in
-   order to fit your needs. Please find attached one file that you
-   can edit it with your favorite text editor.
+   only restriction is that the files must be uncompressed. You can
+   take an existing MetaImage reader and modify it in order to fit
```

Comment:
reader -> header?

Reply:
:eyes: Thanks Dženan ! Changed in 4deead1.